python/src/QuantileMatchingFactory_doc.i.in

```diff
@@ -0,0 +1,177 @@
+%feature("docstring") OT::QuantileMatchingFactory
+"Estimation by matching quantiles.
+
+Implements generic estimation by matching quantiles.
+
+Parameters
+----------
+distribution : :class:`~openturns.Distribution`
+    The distribution defining the parametric model to be adjusted to data.
+    Its parameters define the starting point of the algorithm.
+alpha : sequence of float, optional
+    The quantile levels :math:`\alpha_i`
```

Comment:
Since this parameter is optional, the default value must be presented.
```suggestion
    The quantile levels :math:`\alpha_i`. The default value of the percentile ranks is defined as follows. We define the interval :math:`[\epsilon, 1 - \epsilon]` where the value of :math:`\epsilon$ is defined by the `"QuantileMatchingFactory-QuantileEpsilon"` key of the `:class:`~openturns.ResourceMap`. A regular grid :math:`\{\alpha_i\}_{i = 1, ..., n}` of percentile ranks is defined in the interval :math:`[\epsilon, 1 - \epsilon]`, where :math:`n` is the number of parameters of the distribution. The grid is defined by the equation :math:`\alpha_i = (1 - \theta_i) \epsilon + \theta_i (1 - \epsilon)`
where :math:`\theta_i = \frac{i - 1}{n - 1}` for :math:`i = 1, ..., n`. 
```
This should produce something like this:
![image](https://user-images.githubusercontent.com/31351465/182339515-bae027a3-dc4f-43c0-881d-48bb40586253.png)

The code must be updated accordingly.